aviary/variable_info/variable_meta_data.py

```diff
@@ -1204,9 +1204,35 @@
                      "LEAPS1": None
                      },
     option=True,
+    units="lbm",
+    desc='required fuel reserves: directly in lbm',
+    default_value=0,
+)
+
+add_meta_data(
+    Aircraft.Design.RESERVES_FRACTION,
+    meta_data=_MetaData,
+    historical_name={"GASP": 'INGASP.FRESF2',  # the real name is INGASP.FRESF
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    option=True,
+    units="unitless",
+    desc='required fuel reserves: given as a proportion of mission fuel',
+    default_value=0,
+)
+
+add_meta_data(
+    Aircraft.Design.RESERVES_OPTION,
+    meta_data=_MetaData,
+    historical_name={"GASP": 'INGASP.FRESF3',  # the real name is INGASP.FRESF
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    option=True,
+    types=int,
```

Comment:
Again, I would like to see this be an enum or string instead of int

Reply:
Done.